src/components/v2/V2Create/tabs/OthersTabContent.tsx

```diff
@@ -0,0 +1,130 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, Form, Space, Switch } from 'antd'
+import { useForm } from 'antd/lib/form/Form'
+
+import { useAppDispatch } from 'hooks/AppDispatch'
+import { useAppSelector } from 'hooks/AppSelector'
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import { editingV2ProjectActions } from 'redux/slices/editingV2Project'
+
+import { FormItems } from 'components/shared/formItems'
+import { ThemeContext } from 'contexts/themeContext'
+
+import { ballotStrategies } from 'constants/ballotStrategies/ballotStrategies'
+import { shadowCard } from 'constants/styles/shadowCard'
+
+export type OthersFormFields = {
+  pausePay: boolean
+  pauseMint: boolean
+  ballot: string
+}
+
+export default function OthersTabContent() {
+  const { theme } = useContext(ThemeContext)
+
+  const [othersForm] = useForm<OthersFormFields>()
+  const dispatch = useAppDispatch()
+  const { fundingCycleMetadata, fundingCycleData } = useAppSelector(
+    state => state.editingV2Project,
+  )
+
+  const [showMintingWarning, setShowMintingWarning] = useState<boolean>(false)
+
+  const onOthersFormSaved = useCallback(() => {
+    const fields = othersForm.getFieldsValue(true)
+    console.log('saving fields: ', fields)
+    dispatch(editingV2ProjectActions.setPausePay(fields.pausePay))
+    dispatch(editingV2ProjectActions.setPauseMint(fields.pauseMint))
+    dispatch(editingV2ProjectActions.setBallot(fields.ballot))
+  }, [dispatch, othersForm])
+
+  const resetOthersForm = useCallback(() => {
+    othersForm.setFieldsValue({
+      pausePay: fundingCycleMetadata?.pausePay ?? false,
+      pauseMint: fundingCycleMetadata?.pauseMint ?? false,
+      ballot: fundingCycleData?.ballot ?? ballotStrategies()[2].address, // 3-day delay default
+    })
+  }, [
+    fundingCycleMetadata?.pausePay,
+    fundingCycleMetadata?.pauseMint,
+    fundingCycleData?.ballot,
+    othersForm,
+  ])
+
+  // initially fill form with any existing redux state
+  useEffect(() => {
+    resetOthersForm()
+  }, [resetOthersForm])
+
+  return (
+    <div>
+      <Space direction="vertical" size="large">
+        <p>
+          <Trans>
+            <strong>Note: </strong>Updates you make to attributes in this
+            section will <i>not be applied immediately</i> and only take effect
+            in <i>upcoming funding cycles.</i>
```

Comment:
Is this true for project creation?

Reply:
Updated: `<strong>Note: </strong>Once your project is create, updates you make to attributes in this
            section will <i>not be applied immediately</i> and only take effect
            in <i>upcoming funding cycles.</i>`